gym-socialgame/gym_socialgame/envs/socialgame_env.py

```diff
@@ -328,7 +336,12 @@ def _get_reward(self, price, energy_consumptions, reward_function = "log_cost_re
                     print("Reward function not recognized")
                     raise AssertionError
 
-                total_reward += reward
+                smirl_weight = 0.03
```

Comment:
Only significant change from main?

Reply:
This should prob be a raw weight, not a percentage type thing, so the reward is just the previous rewards `+ weight * smirl_weight`

Reply:
OKay sg

Reply:
yeah I double checked with the paper, somehow I missed that. But it makes sense logically too since the smirl_reward isn't being scaled by the actual reward anyways.